docs/config.md

```diff
@@ -0,0 +1,64 @@
+# DefraDB configuration (YAML)
+
+The default DefraDB directory is `$HOME/.defradb`. It can be changed via the --rootdir CLI flag.
+
+Relative paths are interpreted as being rooted in the DefraDB directory.
+
+## `datastore.store`
+
+Store can be badger or memory. Defaults to `badger`.
+
+- badger: fast pure Go key-value store optimized for SSDs (https://github.com/dgraph-io/badger)
+- memory: in-memory version of badger
+
+## `datastore.maxtxnretries`
+
+Maximum number of times to retry a failed transaction. Defaults to `5`.
```

Comment:
todo: I don't think this line fully explains this option, from my memory this is for P2P only txn commit attempts, but that is not clear here.

Reply:
I've updated it with a bit more text from the db documentation.

Reply:
Cheers, new text looks good.